cfme/middleware/providers.py

```diff
@@ -0,0 +1,32 @@
+from cfme.fixtures import pytest_selenium as sel
+from cfme.web_ui.menu import nav
+from . import list_tbl
+
+
+nav.add_branch(
+    'middleware_providers',
+    {
+        'middleware_provider_detail':
+            lambda ctx: list_tbl.click_cells({'name': ctx['provider'].name})
+    }
+)
+
+
+class Providers(object):
```

Comment:
Is this a different class than the "HawkularProvider" class? What does it represent?